W-10796831: Update docs (#111)
mulesoft/docs-dataweave: modules/ROOT/pages/dw-dates-functions-localtime.adoc

[%header, cols="1,3"]
|===
| Name | Description
| `parts` | `Number` values for `hour`, `minutes`, and
            `seconds` fields. Valid values are 0 through 23 for the

>             `hour`, 0 through 59 for `minutes`, and 0 through 59 (not inclusive; the field accepts 59.99, for example) for `seconds` fields.

Reviewer: TODO: CHECK "not inclusive" here, too. # 3 instance

Reviewer: FIXED

>             `hour`, 0 through 59 for `minutes`, and 0 through 59 (not inclusive; the field accepts 59.99, for example) for `seconds` fields.

Reviewer: ```suggestion
            `hour`, 0 through 59 for `minutes`, and 0 through 59 (including decimals, such as 59.99) for `seconds` fields.
```

Reviewer: fixed in source, master
